src/components/project/project.service.ts

```diff
@@ -526,6 +532,7 @@ export class ProjectService {
       ])
       .asResult<
         StandardReadResult<DbPropsOfDto<Project>> & {
+          pinnedRel: Relation;
```

Comment:
It's an optional match so the relation returned is nullable
```suggestion
          pinnedRel?: Relation;
```